package/gluon-mesh-vpn-wireguard/luasrc/lib/gluon/upgrade/399-mesh-vpn-wireguard-from-fastd

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/lua
```

Comment:
Rename file to 399-mesh-vpn-fastd-wireguard-key-convert

Reply:
resolved in d0f3f919560feb70d4528973e30e3dc06b66d725